app/views/user_mailer/email_changed.html.slim

```diff
@@ -12,4 +12,5 @@ table.hr
 
 p == t('.help',
       app: link_to(APP_NAME, Figaro.env.mailer_domain_name, class: 'gray'),
-      contact_phone_number: CONTACT_PHONE_NUMBER)
+      help_link: link_to(t('user_mailer.help_link_text'), MarketingSite.help_url),
+      contact_link: link_to(t('user_mailer.contact_link_text'), MarketingSite.contact_url))
```

Comment:
Looks like we're making this same change in multiple places. What do you think about moving this to a partial to make it easy to change the next time we'll need to change this?